packages/matchbox/src/components/Button/Button.js

```diff
@@ -137,6 +145,7 @@ Button.propTypes = {
   flat: PropTypes.bool,
   plain: deprecate(PropTypes.bool, 'Use `flat` instead'),
   outline: PropTypes.bool,
+  outlineBorder: PropTypes.bool,
```

Comment:
This is now the fourth boolean prop to determine the visualWeight.  I think the interface for this could be consolidated into a single prop that only accepts values 'flat', 'outline', or 'outlineBorder'.